src/app/tags/page.tsx

```diff
@@ -1,20 +1,52 @@
 import { getPostMetadata } from "@/helper/getPostMetadata";
 import Link from "next/link";
 
-const page = () => {
-  const data = getPostMetadata();
-  const tags = data.flatMap((e) => e.tags.flat((tag: any) => tag));
-  const finalTags = tags.reduce((acc: any, curr: any) => {
-    acc[curr] = (acc[curr] || 0) + 1;
-    return acc;
-  }, {});
+type searchProps = {
+  searchParams?: {
+    sort?: string;
+  };
+};
+
+export const dynamic = "force-dynamic";
```

Comment:
myabe try removing this line, in build this page is rendered as serverless function but instead should be satitically generated like all other routes

Reply:
still does not work

Reply:
https://github.com/vercel/next.js/issues/43077 pls have a look at this i did everything related to the issue but problem seems to be the new /app directory

Reply:
okk so i did a through invastigation, and everything you did seem to be correct one last thing you can still try is to add  ` (const sortType = searchParams?.sort || "alphabet")` circular brackets, if that also does not work just create a normal pr wihout the search param feature 
@sirish123  thanks 👍